### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by valentunn**
* Update ramp staging
* Update bottom navigation style
* wallet ui improvements
* Create separate table for tokens
* Add different title for dev builds
* Fix crash while opening the account actions on profile screen
* Change dead recipient from warning to error
* [Fix] New day is not shown after adding transaction to the list

### CODE REVIEW

File: feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/TransferValidityChecks.kt [hunk 1]
```diff
@@ -55,11 +55,13 @@ private fun BaseFragment<*>.showTransferError(
     status: TransferValidityLevel.Error.Status,
     errorConfirmed: DialogClickHandler?
 ) {
-    val messageRes = when (status) {
-        TransferValidityLevel.Error.Status.NotEnoughFunds -> R.string.choose_amount_error_too_big
+    val (titleRes, messageRes) = when (status) {
+        TransferValidityLevel.Error.Status.NotEnoughFunds -> R.string.common_error_general_title to R.string.choose_amount_error_too_big
+        TransferValidityLevel.Error.Status.DeadRecipient -> R.string.wallet_send_dead_recipient_title to R.string.wallet_send_dead_recipient_message
```
Comment (antonkhvorov) [inline feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/TransferValidityChecks.kt:60]: rip

Comment (antonkhvorov) [reply feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/TransferValidityChecks.kt:60]: good thing that they can be revived by relevantly small price, truly the future we all want

**Review by antonkhvorov — APPROVED** [review]

File: feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/amount/ChooseAmountFragment.kt [hunk 2]
```diff
@@ -90,10 +90,10 @@ class ChooseAmountFragment : BaseFragment<ChooseAmountViewModel>() {
         }
 
         viewModel.assetLiveData.observe {
-            chooseAmountBalance.text = it.available.formatAsToken(it.token)
+            chooseAmountBalance.text = it.available.formatAsToken(it.token.type)
```
Comment (arvifox) [inline feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/amount/ChooseAmountFragment.kt:93]: какие планы про kotlin synthetic? он же deprecated

Comment (valentunn) [reply feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/send/amount/ChooseAmountFragment.kt:93]: Да, они deprecated, но по моему мнению - всё-равно удобнее, чем ViewBinding (нет дополнительного кода для инициализации ViewBinding'ов

**Pull request merged**